lib/std/os.zig

```diff
@@ -7056,3 +7056,51 @@ pub fn timerfd_gettime(fd: i32) TimerFdGetError!linux.itimerspec {
         else => |err| return unexpectedErrno(err),
     };
 }
+
+/// Whether or not the current target support SIGPIPE
+pub const have_sigpipe_support = switch (builtin.os.tag) {
+    .linux,
+    .macos,
+    .netbsd,
+    .solaris,
+    .freebsd,
+    .openbsd,
+    => true,
+    else => false,
+};
+
+pub const keep_sigpipe: bool = if (@hasDecl(root, "keep_sigpipe"))
```

Comment:
This should use the new std.options mechanism: https://github.com/ziglang/zig/blob/c32171991b25b323cd68ff96c294bf5a6fa753b8/lib/std/std.zig#L106